src/Analyzers/Core/CodeFixes/MatchFolderAndNamespace/AbstractChangeNamespaceToMatchFolderCodeFixProvider.CustomFixAllProvider.cs

```diff
@@ -70,7 +70,7 @@ static async Task<ImmutableArray<Diagnostic>> GetSolutionDiagnosticsAsync(FixAll
         private static async Task<Solution> FixAllByDocumentAsync(
             Solution solution,
             ImmutableArray<Diagnostic> diagnostics,
-            IProgress<CodeAnalysisProgress> progressTracker,
+            IProgress<CodeAnalysisProgressWrapper> progressTracker,
```

Comment:
This is also a block. We explicitly made this API public so it could be consumed and so we wouldn't be using some other mechanism.